composer.json

```diff
@@ -19,14 +29,21 @@
         "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
         "sirbrillig/phpcs-variable-analysis": "^2.0",
         "wimg/php-compatibility": "^8.0",
-        "wp-coding-standards/wpcs": "^0.14.0"
+        "wp-coding-standards/wpcs": "^0.14.0",
+        "phpunit/phpunit": "^6.2",
+        "antecedent/patchwork": "2.0.8",
+        "brain/monkey": "^2.0",
+        "mockery/mockery": "~0.9",
+        "wpfulcrum/extender": "^3.0"
     },
     "config": {
         "sort-order": true
     },
     "scripts": {
         "install-codestandards": [
             "Dealerdirect\\Composer\\Plugin\\Installers\\PHPCodeSniffer\\Plugin::run"
-        ]
+        ],
+        "cs-check": "vendor/bin/phpcs --colors",
+        "test": "vendor/bin/phpunit --colors"
```

Comment:
This colors arg can be specified in the `phpunit.xml.dist` file. See https://github.com/GaryJones/plugin-boilerplate/blob/bd9ca02455f5fc2f3b57ade0459d672b9472d392/phpunit.xml.dist#L10